### ISSUE TITLE
docs/nbproject/ in tar.gz off v4.0.0-beta2 release

### ISSUE THREAD

**Issue by dzc34** [Asqatasun/Asqatasun#29]

**Comment by mfaure** [comment]
Having an `nbproject/` directory is not a problem, but a facility for other developers that may be using Netbeans.

However, providing the content of `nbproject/private/`is an issue, that is now fixed by 5cfdbc73f2dc9287ce9071978d67ad4ed2917b5f :)

**Comment by dzc34** [comment]
I don't understand the "facility for other developers that may be using Netbeans"
when I open this files in the tar.gz :


#### docs/nbproject/project.properties   
```
include.path=${php.global.include.path}
php.version=PHP_54
source.encoding=UTF-8
src.dir=.
tags.asp=false
tags.short=false
web.root=.
```
#### docs/nbproject/project.xml
```xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://www.netbeans.org/ns/project/1">
    <type>org.netbeans.modules.php.project</type>
    <configuration>
        <data xmlns="http://www.netbeans.org/ns/php-project/1">
            <name>asqatasun-docs</name>
        </data>
    </configuration>
</project>
```

**Comment by mfaure** [comment]
OK you're right, I mismatched .tar.gz and code itself.
This is now (really) fixed with 46bf9ba256c37c1fdd5b1f5778ac54ae9475a488
